### ISSUE TITLE
cannot install DLC on Ubuntu 18.04 or 22.04

### ISSUE THREAD

**Issue by huguenardlab** [DeepLabCut/DeepLabCut#2174]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

We are trying to install the latest version of DLC on Ubuntu with an Nvidia GPU, and cannot seem to find a version match that works for all of tensorflow, cuda, cudnn, and DLC.

We have a simple question that does not seem to be answered anywhere on the forum, which is this:

For a clean Ubuntu install with GPU, what version of Ubuntu do you recommend, with this clean install, what versions of the nvidia driver and cuda do you recommend?

Has anyone successfully installed DLC 2.3 on a new Ubuntu 22.04 LTS system with GPU suppport?  Can you share your experience and settings.

### Operating System

operating system Ubuntu 18.04 or 22.04 or 20.04

### DeepLabCut version

dlc version 2.3

### DeepLabCut mode

single animal

### Device type

gpu Tesla P100

### Steps To Reproduce

We get different compatibility issues with each different version of ubuntu.  For example by downgrading to 18.04 we don't seem to be able to obtain a libcudnn version from Nvidia that is appropriate (8.0).  This is just one of many examples.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
You should use version 2.3.1, which pins pytorch and solves the issue. I use ubuntu 22.04 without issue; my install guide is here: https://deeplabcut.github.io/DeepLabCut/docs/recipes/installTips.html#installation-on-ubuntu-20-04-lts

let me know if that solves the issue for you // will close since answered though 👍

**Comment by huguenardlab** [comment]
Thanks!  We now have the docker version working well, with GPU support, thanks to the guidance provided on this site and @tbenst and on Nvidia's docker support page.  When we used a pip install of deeplabcut, which defaulted to 2.3.2 it tried to start up tensorflow, and failed.  When we did pip install deeplabcut==2.3.1 it did downgrade to the version you recommended but it still seems to be looking for tensorflow and failed.  Is there a configuration setting I am missing or an alternate way of downloading dlc that I am missing that will get us the pytorch enabled version?
